### PR TITLE
Update catalog.md

### PR DESCRIPTION
**Pull request by lzbj**
Address #364.
Add the documentation for catalog to reflect the latest action added for text2Voice.

### CODE REVIEW

**Comment by rabbah** [comment]
@csantanapr is there bluemix-docs pr?

File: docs/catalog.md [hunk 1]
```diff
@@ -370,6 +370,38 @@ Here is an example of creating a package binding and identifying the language of
   }
   ```
 
+### Synthesizes text input to spoken audio
+
+The `/whisk.system/watson/textToSpeech` action help synthesize text input of any supported language to produce audio output. The parameters are as follows:
```
Comment (rabbah) [inline docs/catalog.md]: action help synthesize text input of any supported language -> action synthesizes text input from a supported language

File: docs/catalog.md [hunk 2]
```diff
@@ -370,6 +370,38 @@ Here is an example of creating a package binding and identifying the language of
...
+
+- `username`: The Watson API username.
+- `password`: The Watson API password.
+- `payload`: The text to be synthesized.
+- `voice`: The voice to be used for synthesis.
+- `accept`: The requested MIME type of the audio.
+- `encoding`: The encoding of the speech binary data.
+
+Here is an example of creating a package binding and synthesize the audio of some input text.
```
Comment (rabbah) [inline docs/catalog.md]: Here is an example of creating a package binding and synthesize the audio of some input text. ->
The following example creates a Watson text to speech package binding to synthesize input text to audio.

File: docs/catalog.md [hunk 3]
```diff
@@ -370,6 +370,38 @@ Here is an example of creating a package binding and identifying the language of
...
+
+1. Create a package binding with your Watson credentials.
+
+  ```
+  $ wsk package bind /whisk.system/watson myWatson -p username 'MY_WATSON_USERNAME' -p password 'MY_WATSON_PASSWORD'
+  ```
+
+2. Invoke the `textToSpeech` action in your package binding to identify the language.
```
Comment (rabbah) [inline docs/catalog.md]: identify the language? Guess this is cut and pasted from another Watson action.

**Comment by jasonpet** [comment]
There was a docs pr done for this a while back by Joachim:
https://github.com/IBM-Bluemix/docs/pull/169

**Comment by csantanapr** [comment]
Hi @lzbj Thanks for the pull request, sorry that we can't take the PR, but there is documentation work already in place like @jasonpet commented here https://github.com/openwhisk/openwhisk/pull/488#issuecomment-221016247

I'm going to close it, and we are going to sync that content from in this repo, you can track progress in #364

**Comment by lzbj** [comment]
@csantanapr , no worry, glad to know it, next time I'll confirm if there is some overlap before I continue some works on documentation etc.
@rabbah , thanks a lot for your review, I'll make sure before submit pull request, to avoid waste your time.

**Pull request closed without merging**